server/public/pluginapi/system.go

```diff
@@ -17,6 +20,10 @@ type SystemService struct {
 	api plugin.API
 }
 
+const (
+	PacketMetadataFileName = "packet_metadata.yaml"
```

Comment:
@ifoukarakis, do you have a strong option on how the file should be named?

Reply:
Not really, as long as it doesn't conflict with other well-known files appearing in support packages. `metadata.json` and `metadata.yaml` were considered so far.

Reply:
Then let's go with `metadata.yaml`.